<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/Builder.png</normaloff>:/res/Builder.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
			   background-color: rgb(255, 255, 255);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
              }
QMenuBar {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QStatusBar {
               color: rgb(32, 74, 135);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>32</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenu {
                background-color: #fff;
                border: none;
                border-radius: 0px;
                }
                QMenu::item {
                padding-left: 18px;
                margin: 4px;
                }
                QMenu::item:selected {
                color: #111;
                background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                        stop: 0 #fff, stop: 1 #eef);
                }
                QMenu::separator {
                height: .5px;
                background: #eee;
                margin-left: 10px;
                margin-right: 5px;
                }
                QMenu::indicator {
                width: 14px;
                height: 14px;
                }
                QMenu::indicator:non-exclusive:checked {
                image: url(:/pict/check.png);
                }
                QMenu::indicator:non-exclusive:unchecked {
                image: url(:/pict/unchecked.png);
           }</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuAccount_settings">
    <property name="title">
     <string>Account settings</string>
    </property>
    <addaction name="actionGroups"/>
    <addaction name="actionUsers"/>
    <addaction name="actionTenant"/>
    <addaction name="actionLogo"/>
    <addaction name="actionSecurity_filters"/>
   </widget>
   <widget class="QMenu" name="menuScreen_configurations">
    <property name="title">
     <string>Screen configurations</string>
    </property>
    <addaction name="actionMenu"/>
    <addaction name="actionGroup_screens"/>
    <addaction name="actionScreens"/>
   </widget>
   <widget class="QMenu" name="menuDocument">
    <property name="title">
     <string>Document access</string>
    </property>
    <addaction name="actionDocument_family"/>
    <addaction name="actionDocument_groups"/>
   </widget>
   <widget class="QMenu" name="menuSystem_data">
    <property name="title">
     <string>System data</string>
    </property>
    <addaction name="actionLists"/>
    <addaction name="actionTemplates"/>
    <addaction name="actionCustom_data_sourcess"/>
    <addaction name="actionExtention_functions"/>
    <addaction name="actionServers"/>
    <addaction name="actionSecuriry_questions"/>
   </widget>
   <widget class="QMenu" name="menuFeatures">
    <property name="title">
     <string>Features</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/features.png</normaloff>:/res/features.png</iconset>
    </property>
    <addaction name="actionCustom_query"/>
    <addaction name="actionManagement"/>
    <addaction name="actionImport_files"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuAccount_settings"/>
   <addaction name="menuScreen_configurations"/>
   <addaction name="menuDocument"/>
   <addaction name="menuSystem_data"/>
   <addaction name="menuFeatures"/>
  </widget>
  <action name="actionGroups">
   <property name="text">
    <string>Groups</string>
   </property>
  </action>
  <action name="actionUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionTenant">
   <property name="text">
    <string>Tenant</string>
   </property>
  </action>
  <action name="actionLogo">
   <property name="text">
    <string>Logo</string>
   </property>
  </action>
  <action name="actionSecurity_filters">
   <property name="text">
    <string>Security filters</string>
   </property>
  </action>
  <action name="actionMenu">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="actionGroup_screens">
   <property name="text">
    <string>Group screens</string>
   </property>
  </action>
  <action name="actionScreens">
   <property name="text">
    <string>Screens</string>
   </property>
  </action>
  <action name="actionDocument_family">
   <property name="text">
    <string>Document family</string>
   </property>
  </action>
  <action name="actionDocument_groups">
   <property name="text">
    <string>Document groups</string>
   </property>
  </action>
  <action name="actionLists">
   <property name="text">
    <string>Lists</string>
   </property>
  </action>
  <action name="actionTemplates">
   <property name="text">
    <string>Templates</string>
   </property>
  </action>
  <action name="actionCustom_data_sourcess">
   <property name="text">
    <string>Custom data sources</string>
   </property>
  </action>
  <action name="actionExtention_functions">
   <property name="text">
    <string>Extestion functions</string>
   </property>
  </action>
  <action name="actionServers">
   <property name="text">
    <string>Servers</string>
   </property>
  </action>
  <action name="actionSecuriry_questions">
   <property name="text">
    <string>Security questions</string>
   </property>
  </action>
  <action name="actionCustom_query">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/configure.png</normaloff>:/res/configure.png</iconset>
   </property>
   <property name="text">
    <string>Custom query</string>
   </property>
  </action>
  <action name="actionManagement">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/view.png</normaloff>:/res/view.png</iconset>
   </property>
   <property name="text">
    <string>Management</string>
   </property>
  </action>
  <action name="actionImport_files">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/download.png</normaloff>:/res/download.png</iconset>
   </property>
   <property name="text">
    <string>Import files</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/log.png</normaloff>:/res/log.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
